<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="selectAttr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	color:#000;
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #d0d0d0;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 6px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Coordinates</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>28</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Style:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-9</x>
     <y>60</y>
     <width>1081</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="simpleAttrGraph">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QComboBox" name="selectStyle">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	color: #000;
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #d0d0d0;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 0px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>28</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Attribute:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
